Sources/Datadog/FeaturesIntegration/TracingWithLoggingIntegration.swift

```diff
@@ -80,20 +73,26 @@ internal struct TracingWithLoggingIntegration {
             )
         }
 
-        let log = logBuilder.createLogWith(
-            level: level,
-            message: message,
-            error: extractedError,
-            date: date,
-            attributes: LogEvent.Attributes(
-                userAttributes: userAttributes,
-                internalAttributes: internalAttributes
-            ),
-            tags: []
-        )
+        let threadName = getCurrentThreadName()
+
+        self.core.v1.scope(for: LoggingFeature.self)?.eventWriteContext { context, writer in
+            let log = logBuilder.createLogEvent(
+                date: date,
+                status: level.asLogStatus,
+                message: message,
+                error: extractedError,
+                attributes: .init(
+                    userAttributes: userAttributes,
+                    internalAttributes: internalAttributes
+                ),
+                tags: [],
+                context: context,
+                threadName: threadName
+            )
 
-        if let event = log {
-            loggingOutput.write(log: event)
+            if let log = log {
+                writer.write(value: log)
+            }
         }
```

Comment:
Here we switch from writing to `LogOutput` to using V2's _event write context_. The way I see it is that soon "tracing with logging" integration will be done through event bus and the final `LogEvent` write will use _event write context_. In that sense it's a step forward, WDYT @maxep ?

The reason of this is the change in `logBuilder.createLogWith()` which makes it easier to use in V2, not V1.

Reply:
Sounds great 👍